docs/java_cli_for_managedgateway.rst

```diff
@@ -145,8 +145,8 @@ The gateway can invoke sendDeviceManageRequest() method to make the attached dev
 
 .. code:: java
 
-	managedGateway.sendGatewayManageRequest(typeId, deviceId, lifetime, true, true);
-	
+	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);
```

Comment:
It's this bit in bold below that implies that you don't need the lifetime, true true part in the code sample.

If not, then it would be better to rephrase the sentence. Can you explain this a bit more so that I can rephrase?
Thanks


"The gateway can invoke sendDeviceManageRequest() method to make the attached devices participate in the device management activities. 

.. code:: java

	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);

**_As shown, this method accepts the details of the attached device apart from the lifetime and device/firmware support parameters._** The gateway can also use the overloaded sendDeviceManageRequest() method to specify the DeviceData for the attached device.
"